src/components/NavTree/NavTree.tsx

```diff
@@ -19,6 +19,7 @@ export interface NavItemProps extends NavItemConfig {
   // The following props don't need to be explicitly defined - they are passed internally
   onOpen?: () => void;
   level: number;
+  styleProps?: any;
```

Comment:
Exposing a `styleProps` prop is inconsistent with how Chakra handles styles. To avoid over-complicating an alternative solution, I think it would make a lot of sense to expose a `variant` prop, which would set the necessary styles internally

You can also reference the [chakra component styling doc](https://chakra-ui.com/docs/theming/component-style) if you want to implement this the "chakra way"

Reply:
Gotcha, yeah I wasn't too sure about this. I've switched it out with a "variant" field 👍